src/modules/assessments/assessmentColumns.tsx

```diff
@@ -0,0 +1,44 @@
+import { ClientAssessmentType } from '@/components/clientDashboard/enrollments/ClientAssessments';
+import { ColumnDef } from '@/components/elements/table/types';
+import { HhmAssessmentType } from '@/modules/enrollment/components/HouseholdAssessmentsTable';
+import AssessmentDateWithStatusIndicator from '@/modules/hmis/components/AssessmentDateWithStatusIndicator';
+import EnrollmentDateRangeWithStatus from '@/modules/hmis/components/EnrollmentDateRangeWithStatus';
+import { formRoleDisplay, lastUpdatedBy } from '@/modules/hmis/hmisUtil';
+import { ProjectAssessmentType } from '@/modules/projects/components/ProjectAssessments';
+import { AssessmentFieldsFragment } from '@/types/gqlTypes';
+
+export const ASSESSMENT_COLUMNS: {
+  [key: string]: ColumnDef<
+    | ProjectAssessmentType
+    | ClientAssessmentType
+    | HhmAssessmentType
+    | AssessmentFieldsFragment
+  >;
+} = {
+  date: {
+    header: 'Assessment Date',
+    render: (a) => <AssessmentDateWithStatusIndicator assessment={a} />,
+  },
+  type: {
+    header: 'Assessment Type',
+    render: (a) => formRoleDisplay(a),
+  },
+  linkedType: {
```

Comment:
I did some cleanup here to follow the pattern from e.g. CLIENT_COLUMNS which I remember discussing on some recent PR. I wanted to check what you think about this `linkedType` column. It's the easiest but probably least sophisticated way to deal with the problem of different assessment tables having slightly different requirements (the 'type' column values should have link treatment on some tables, but not on others). Maybe there is a better/prettier way to do this?

Reply:
This seems fine to me!